src/components/Display/Display.text

```diff
@@ -0,0 +1,3 @@
+Display 
+
+prop => value must be a string; 
```

Comment:
Some other suggestions for display might be, 'it contains a wrapper div', or 'it should display a value' - general things like this.